README.md

```diff
@@ -198,3 +198,19 @@ If you have to know the version installed for other steps of your workflow you c
 
 [<img src="https://raw.githubusercontent.com/astral-sh/setup-uv/main/docs/images/bmc-button.svg" width=150 height=40 style="margin: 5px"/>](https://www.buymeacoffee.com/eifinger)
 [<img src="https://raw.githubusercontent.com/astral-sh/setup-uv/main/docs/images/paypal-button.svg" width=150 height=40 style="margin: 5px"/>](https://paypal.me/kevinstillhammer)
```

Comment:
Should remove these right?

<img width="836" alt="Screenshot 2024-09-04 at 4 04 55 PM" src="https://github.com/user-attachments/assets/cea2f842-b9b3-415b-a579-2995d8092986">